src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -261,8 +267,8 @@ class DoujinDesu : ParsedHttpSource() {
         }
     }
 
-    private fun getNumberFromString(epsStr: String): String {
-        return epsStr.filter { it.isDigit() }
+    private fun getNumberFromString(epsStr: String?): Float {
+        return epsStr?.filter { it.isDigit() }?.toFloatOrNull() ?: 1F
```

Comment:
Should instead default to `-1f`.

```suggestion
        return epsStr?.filter { it.isDigit() }?.toFloatOrNull() ?: -1f
```